cfme/tests/test_rest.py

```diff
@@ -130,17 +129,18 @@ def test_query_simple_collections(appliance, collection_name):
 @pytest.mark.uncollectif(
     lambda collection_name:
         (collection_name in COLLECTIONS_OMMITED) or
-        (collection_name in COLLECTIONS_ADDED_IN_57 and current_version() < "5.7") or
         (collection_name in COLLECTIONS_ADDED_IN_58 and current_version() < "5.8") or
-        (collection_name in COLLECTIONS_BUGGY_ATTRS and
-            BZ(1437201, forced_streams=["5.6", "5.7", "5.8", "upstream"]).blocks)
+        (collection_name in COLLECTIONS_REMOVED_IN_59 and current_version() >= "5.9")
 )
+@pytest.mark.meta(blockers=['GH#ManageIQ/manageiq:15754'])
 def test_select_attributes(appliance, collection_name):
     """Tests that it's possible to limit returned attributes.
 
     Metadata:
         test_flag: rest
     """
+    if collection_name in COLLECTIONS_BUGGY_ATTRS and current_version() < '5.9':
+        pytest.skip("Affected by BZ 1437201, cannot test.")
```

Comment:
As I know skipped tests affect on successful pass score in Jenkins, maybe it better to uncollect them here?